libreflect/src/stack_setup.c

```diff
@@ -44,7 +44,7 @@ void load_program_info(size_t *auxv, ElfW(Ehdr) *exe, ElfW(Ehdr) *interp)
 	int ii;
 	size_t exe_loc = (size_t) exe, interp_loc = (size_t) interp;
 
-	for (ii = 0; auxv[ii]; ii += 2) {
+	for (ii = 0; auxv[ii] || auxv[ii + 1]; ii += 2) {
```

Comment:
Since it's just `AT_NULL` signalling the end of the vector, this change does not make sense
```suggestion
	for (ii = 0; auxv[ii]; ii += 2) {
```